plugins/source-terasology-engine/gatsby-node.js

```diff
@@ -0,0 +1,103 @@
+const { graphql } = require("@octokit/graphql");
+const { DateTime } = require("luxon");
+const fs = require("fs");
+
+const PLUGIN_NAME = "source-terasology-engine";
+
+const query = `
+query Engine {
+  organization(login: "MovingBlocks") {
+    repository(name: "Terasology") {
+      issues(first: 10, filterBy: {labels: "Good First Issue", states: OPEN}, orderBy: {field: UPDATED_AT, direction: DESC}) {
```

Comment:
Why are we only fetching the first ten issues here? This seems to be quite a few, and even if we fetch more we can still decide to only display a few on the page...

Reply:
Let's change this to fetching all of them with respective pagination in a follow-up PR